config/manager/manager.yaml

```diff
@@ -3,67 +3,95 @@ kind: Namespace
 metadata:
   labels:
     control-plane: controller-manager
-  name: node-feature-discovery-operator
+    app.kubernetes.io/name: namespace
+    app.kubernetes.io/instance: system
+    app.kubernetes.io/component: manager
+    app.kubernetes.io/created-by: node-feature-discovery-operator
+    app.kubernetes.io/part-of: node-feature-discovery-operator
+    app.kubernetes.io/managed-by: kustomize
+  name: system
 ---
 apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: controller-manager
-  namespace: node-feature-discovery-operator
+  namespace: system
```

Comment:
Is this desired default ns?

Reply:
that get's override when deploying, but is good to have it point to where we want from the beginning